configure.py

```diff
@@ -1697,7 +1697,7 @@ def MatchingFor(*versions):
     ActorRel(NonMatching, "d_a_obj_ospbox"),
     ActorRel(Matching,    "d_a_obj_paper", extra_cflags=['-pragma "nosyminline on"']),
     ActorRel(MatchingFor("GZLJ01", "GZLE01", "GZLP01"),    "d_a_obj_pbka"),
-    ActorRel(NonMatching, "d_a_obj_pfall"),
+    ActorRel(Matching, "d_a_obj_pfall", extra_cflags=['-pragma "nosyminline on"']),
```

Comment:
```suggestion
    ActorRel(MatchingFor("GZLJ01", "GZLE01", "GZLP01"), "d_a_obj_pfall", extra_cflags=['-pragma "nosyminline on"']),
```
It only matches for retail, so unmark it as matching for the demo version so the build doesn't fail